index.html

```diff
@@ -289,30 +289,27 @@ <h3>Full Name</h3>
             <div class="custom-card-wrapper col-12 col-md-6 col-lg-4">
               <div class="custom-card">
                 <div class="avatar-wrapper">
-                  <img src="img/placeholder.jpeg" alt="" class="avatar" />
+                  <img src="img/PaulaGoras.jpeg" alt="" class="avatar" />
                 </div>
                 <div>
-                  <h3>Full Name</h3>
+                  <h3>Paula Goras</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                   
+                    Sometimes is not easy to be yourself, some people will agree, some won't. But I am not afraid, I am not special like others, I am special like me. 
```

Comment:
`Sometimes is not easy to be yourself` should be `Sometimes it's not easy to be yourself`